cucumber/api/features/support/ssh_helpers.rb

```diff
@@ -6,9 +6,9 @@
 module SshHelpers
 
   def run_command_in_machine(machine_ip, machine_username, machine_password, command)
-    ssh = Net::SSH.start(machine_ip, machine_username, :password => machine_password)
```

Comment:
why was this changed? `:something` is a property of a hash that targets the key. Is NetSSH not expecting a password property?

Reply:
It does expect `password` and still gets it. I just changed the syntax